README.md

```diff
@@ -12,26 +12,35 @@ Recommended practices
 
 - Should be defined in files named as `sql/table_version.sql` e.g.
   `sql/clients_daily_v6.sql`
+- May be generated using a python script that prints the query to stdout
+  - Should save output as `sql/table_version.sql` e.g. `sql/core_clients_daily_v1.sql`
+  - Should be named as `sql/name.sql.py` e.g. `sql/clients_daily.sql.py`
```

Comment:
Can we make these two a bit more parallel? Perhaps:

```suggestion
  - Should be named as `sql/table.sql.py` e.g. `sql/core_clients_daily.sql.py`
```

Do you think we should have a version suffix on the python scripts too?

Reply:
I'm still not sure the best way to name these, so I was being intentionally generic about the naming so we could restrict it later. If we can hash it out here, all the better.

I don't think we should version the scripts, for now, much like we don't version `GenericCountView` or `GenericLongitudinal` in t-b-v.

 For naming, the `Generic` part seems implied in by using a script instead of plain SQL, and `Count` and `Longitudinal` aren't really tables. As such, my counter-suggestion is to refer to it as `query_type` (or perhaps just `type`? idk, feels too vague), where the `core_` prefix is a specific instance of the pattern.

```diff
 - Should be defined in files named as `sql/table_version.sql` e.g.
-  `sql/clients_daily_v6.sql`
+  `sql/core_clients_daily_v6.sql`
 - May be generated using a python script that prints the query to stdout
-  - Should save output as `sql/table_version.sql` e.g. `sql/core_clients_daily_v1.sql`
-  - Should be named as `sql/query_type.sql.py` e.g. `sql/clients_daily.sql.py`
+  - Should save output as `sql/table_version.sql` as above
+  - Should be named as `sql/query_type.sql.py` e.g. `sql/clients_daily.sql.py`
```

Reply:
Ah, I hadn't really thought through the fact that a single script could be used on top of multiple tables to generate multiple different output tables. I like what you've proposed. Perhaps it's worth mentioning here that the scripts may take arguments to configure them for different targets.

Reply:
okay, added a bullet:
  - May use options that configure queries to generate different destination
    tables e.g. using `--source telemetry_core_parquet_v3` to generate
    `sql/core_clients_daily_v1.sql` and using `--source main_summary_v4` to
    generate `sql/clients_daily_v7.sql`